Source/Microsoft.Teams.Apps.GoalTracker/ClientApp/src/i18n.ts

```diff
@@ -7,17 +7,22 @@
 import i18n from "i18next";
 import { initReactI18next } from "react-i18next";
 import Backend from 'i18next-xhr-backend';
+import * as microsoftTeams from "@microsoft/teams-js";
+let locale = "en-US";
+microsoftTeams.initialize();
+microsoftTeams.getContext((context: microsoftTeams.Context) => {
+    i18n.changeLanguage(context.locale!);
```

Comment:
@abbodh Does the UX load in the fallback language, then reload in the new language when the context becomes available?